Sources/SwifterSwift/SwiftStdlib/RangeReplaceableCollectionExtensions.swift

```diff
@@ -174,4 +174,24 @@ public extension RangeReplaceableCollection {
                 with: newValue)
         }
     }
+    
+    /**
+     SwifterSwift: Adds a new element at the end of the array, mutates the array in place
+     - Parameter newElements: The optional element to append to the array
+     */
+    mutating func appendIfNonNil(_ newElement: Self.Element?)
+        {
+            guard let newElement = newElement else {return}
+            self.append(newElement)
+        }
+    
+    /**
+     SwifterSwift: Adds the elements of a sequence to the end of the array, mutates the array in place
+     - Parameter newElements: The optional sequence to append to the array
+     */
+    mutating func appendIfNonNil<S>(contentsOf newElements: S?) where Element == S.Element, S : Sequence
+        {
```

Comment:
Nit: Braces start on the same line, this keeps consistent with the rest of the project =]

Reply:
Also seems like there is one extra tab here ...